docs/distributed/README.md

```diff
@@ -43,6 +43,8 @@ To start a distributed MinIO instance, you just need to pass drive locations as
 - Servers running distributed MinIO instances should be less than 15 minutes apart. You can enable [NTP](http://www.ntp.org/) service as a best practice to ensure same times across servers.
 - Running Distributed MinIO on Windows operating system is experimental. Please proceed with caution.
 - `MINIO_DOMAIN` environment variable should be defined and exported if domain is needed to be set.
+- The number of drives needs to be a multiple of 4, 6, 8, 10, 12, 14 or 16.
```

Comment:
```suggestion
- The number of drives needs to be a multiple of 4, 6, 10 or 14.
```
These imply the others.

Reply:
Actually @klauspost the list is right not sure why are you saying it's 10 or 14 only

Reply:
@harshavardhana 8, 12, 16 are all a multiple of 4.

Reply:
If there is some additional meaning, the text should be updated to that. As it is, it doesn't make sense.

Reply:
Perhaps we should mention it as divisible.